records_mover/records/targets/google_sheets.py

```diff
@@ -104,7 +104,7 @@ def _get_service(self) -> SheetsService:
     def as_json_serializable(self, cell: Any) -> Any:
         if isinstance(cell, np.generic):
             # MyPy complains that this method does not exist
-            native = np.asscalar(cell)  # type: ignore
+            native = cell.item()
```

Comment:
can't add this comment at the exact line but it seems like there should be a way to use numpy.ndarray.tobytes or something similar to serialize this instead of the nested list comprehension calling this method down below.

Reply:
definitely agree; this is actually just a change I pulled forward from the `RM-7-update-default-python-image-for-integration-tests-to-3-9-remove-build-tests-for-deprecated-python-version` branch where there were so many errors on transition that I was trying to keep changes as minimal as possible. 

*But,* if you're willing to dive deep and rewrite this call, just create a ticket for it and it'll be a good way to get to know the tool better! We'll fix the failing tests first but then the world is our oyster. =)